examples/Cassie/cassie_rbt_state_estimator.cc

```diff
@@ -47,33 +62,75 @@ CassieRbtStateEstimator::CassieRbtStateEstimator(
   velocity_index_map_ = multibody::makeNameToVelocitiesMap(tree);
 
   // Initialize body indices
-  left_thigh_ind_ = GetBodyIndexFromName(tree, "thigh_left");
-  right_thigh_ind_ = GetBodyIndexFromName(tree, "thigh_right");
-  left_heel_spring_ind_ = GetBodyIndexFromName(tree, "heel_spring_left");
-  right_heel_spring_ind_ = GetBodyIndexFromName(tree, "heel_spring_right");
-  DRAKE_DEMAND(left_thigh_ind_ != -1 && right_thigh_ind_ != -1 &&
-               left_heel_spring_ind_ != -1 && right_heel_spring_ind_ != -1);
+  left_thigh_idx_ = GetBodyIndexFromName(tree, "thigh_left");
+  right_thigh_idx_ = GetBodyIndexFromName(tree, "thigh_right");
+  left_heel_spring_idx_ = GetBodyIndexFromName(tree, "heel_spring_left");
+  right_heel_spring_idx_ = GetBodyIndexFromName(tree, "heel_spring_right");
+  DRAKE_DEMAND(left_thigh_idx_ != -1 && right_thigh_idx_ != -1 &&
```

Comment:
Although it's good to check that the indices are valid, I feel like this check should be inside GetBodyIndexFromName

Reply:
Yeah I agree. I don't see the use case where we want to continue the program even though the body name doesn't exist.